README.md

```diff
@@ -99,14 +99,17 @@ var indexes = await client.GetAllIndexes();
 ```
 ### Documents
 
-#### Add Documents <!-- omit in toc -->
+#### Add or Update Documents <!-- omit in toc -->
 
 ```c#
- var updateStatus = await index.AddorUpdateDocuments(new[]{new  Movie {Id = "1", Name = "Batman"}});
+ var updateStatus = await index.AddDocuments(new[]{new  Movie {Id = "1", Name = "Batman"}});
+ var updateStatus = await index.UpdateDocuments(new[]{new  Movie {Id = "1", Name = "Batman"}});
```

Comment:
You're right!
In the MeiliSearch API there are:
- the “Add or replace documents” route: it adds a document if it does not exist and overwrite the previous document otherwise
- the “Add or update documents” route: it adds a document if it does not exist and partially update the previous document otherwise (it does not overwrite the previous fields if not mentioned in the body)
Why do we choose to call our method (in the SDKs) `addDocuments` for `addOrReplaceDocuments` and `updateDocuments` for `AddOrUpdateDocuments`? For simplicity. It’s clearer for the user without reading the MeiliSearch docs. You want to add: you use `addDocuments`. You want to update: you use `updateDocuments`. An API has to be accurate in its routes naming. An SDK has to be easy to use since it's a wrapper to help user play and code with MeiliSearch 🚀
That’s why we decided to keep these shorter names 🙂